src/desktop-activity.md

```diff
@@ -225,17 +224,17 @@ activities, available at [Make Your Own Sugar Activities](https://flossmanuals.n
 
 # Coding standards
 
-We currently use [Python Version 2.7](https://docs.python.org/2/) for
+We currently use [Python Version 3.6](https://docs.python.org/3/) for
```

Comment:
Is there a specific need for restricting future development to 3.6, even when 3.7 becomes available?  If not, let's just say `3` here instead of `3.6`.